condor_watch_q.py

```diff
@@ -484,15 +480,23 @@ def watch_q(
                 )
                 msg += [""]
 
+            terminal_rows, terminal_columns = os.popen("stty size", "r").read().split()
+            terminal_columns = int(terminal_columns)
```

Comment:
Let's factor this out into a function. We also need a Windows solution (doesn't have to be in this PR; make an issue for it).